plugin/gcp/storageObjectPrefix.go

```diff
@@ -0,0 +1,42 @@
+package gcp
+
+import (
+	"context"
+
+	"cloud.google.com/go/storage"
+	"github.com/puppetlabs/wash/plugin"
+)
+
+type storageObjectPrefix struct {
+	plugin.EntryBase
+	bucket *storage.BucketHandle
+	prefix string
+}
+
+// Takes the name of the directory, as well as the full prefix path.
+// Attrs may be nil if they could not be retrieved. Some prefixes don't appear to have attributes.
+func newStorageObjectPrefix(bucket *storage.BucketHandle,
+	name, prefix string, attrs *storage.ObjectAttrs) *storageObjectPrefix {
+	pre := &storageObjectPrefix{
+		EntryBase: plugin.NewEntry(name),
+		bucket:    bucket,
+		prefix:    prefix,
+	}
+	if attrs != nil {
+		pre.Attributes().SetMtime(attrs.Updated).SetSize(uint64(attrs.Size)).SetMeta(attrs)
```

Comment:
What does Size mean here? Kind of weird seeing it set w/o implementing Readable.

Reply:
Usually 0. Directories do not universally have size 0 however. https://unix.stackexchange.com/a/77. I would interpret it as the size of the readdir response.